app/models/katello/glue/pulp/repo.rb

```diff
@@ -752,6 +752,14 @@ def importer_matches?(capsule_importer, capsule)
           generated_importer.config == capsule_importer['config']
       end
 
+      def module_streams_data
+        unit_search(type_ids: [ "modulemd" ]).parsed_body
+      end
+
+      def module_defaults_data
+        unit_search(type_ids: [ "modulemd_defaults" ]).parsed_body
```

Comment:
define these type_id strings as a constant somewhere and use that constant. I am wondering if you should add it in runcible.

Reply:
These two methods aren't actually used anywhere and I just had them as helper methods when I was parsing the data. I think its probably best if I remove them for now